internal/pkg/plugin/trellogithub/read.go

```diff
@@ -28,8 +29,13 @@ func IsHealthy(options *map[string]interface{}) (bool, error) {
 		log.Infof("The workflow/file %s is ok", name)
 	}
 	if errFlag {
-		return false, nil
+		return nil, nil
 	}
 
-	return true, nil
+	c, err := trello.NewClient()
+	if err != nil {
+		return nil, err
+	}
+
+	return c.GetBoard()
```

Comment:
"Read" returns state+error, but "GetGoard" doesn't really seem to be related with "state". Maybe we can create a better name.

Some examples (not good but clearer):

1. GetBoardsAndBuildState?
2. boards = getBoards(); return buildState(boards), nil